ckanext/datatablesview/templates/datatables/datatables_view.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block page %}
-  {% set filters = request.params.get('filters', '')|e if request.params is defined else request.args.get('filters', '')|e %}
+  {% set filters = h.get_filters(encode=True) %}
```

Comment:
It looks like you removed the handling of `request.args`. For submitting upstream you'll need both because flask request objects put the parameters in `request.args`

Reply:
@wardi in my helper functions, I use ckan.plugins.toolkit.request, which has it's params property definition as such:
`@property
    def params(self):
        u''' Special case as Pylons' request.params is used all over the place.
        All new code meant to be run just in Flask (eg views) should always
        use request.args
        '''
        try:
            return super(CKANRequest, self).params
        except AttributeError:
            return self.args`
So that should handle any issues between the Flask and Pylons request objects.